### PR TITLE
GSG-546 & GSG-547 Updates

### PR DESCRIPTION
**Pull request by sarah-welton**
# **Please read the [Contributing guide](CONTRIBUTING.md) before making a PR.**

* Read the [Graphics repository & Yamato FAQ](http://go/graphics-yamato-faq).

### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [n/a ] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [x ] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the [Docs team contacts sheet](https://docs.google.com/spreadsheets/d/1rgUWWgwLFEHIQ3Rz-LnK6PAKmbM49DZZ9al4hvnztOo/edit#gid=1058860420).
- [n/a ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

Made changes to About Shader Graph page and README file to satisfy changes needed by [GSG-546](https://jira.unity3d.com/browse/GSG-546) & [GSG-547](https://jira.unity3d.com/browse/GSG-547)

Backports are needed for the changes made to the README file **only** for 7.x.x, 10.x.x, and 2021.2.
Depending on how long merge takes, backport may be needed for About Shader Graph page to 2021.2. 

---
### Testing status
Describe what manual/automated tests were performed for this PR

n/a
---
### Comments to reviewers
Notes for the reviewers you have assigned.

Some text needed to be added to address the changes with SRP2Core, as provided by PMs, and then we had something in the README about the 2018 version of Unity and Shader Graph... I updated the installation instructions in the README, too, just in case anyone looks there :)

### CODE REVIEW

File: com.unity.shadergraph/Documentation~/index.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ Shader Graph enables you to build shaders visually. Instead of writing code, you
...
+Shader Graph is available through the Package Manager window in supported versions of the Unity Editor. If you install a prebuilt Scriptable Render Pipeline (SRP) such as the [Universal Render Pipeline (URP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest) or the [High Definition Render Pipeline (HDRP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest), Unity automatically installs Shader Graph in your project.
 
 Shader Graph package versions on Unity Engine 2018.x are *Preview* versions, which do not receive bug fixes and feature maintenance. To work with an actively supported version of Shader Graph, use Unity Engine 2019.1 or higher.
+
+### SRP packages are part of the core
```
Comment (jessebarker) [inline com.unity.shadergraph/Documentation~/index.md:13]: whitespace at the end of this line

File: com.unity.shadergraph/README.md [hunk 2]
```diff
@@ -4,17 +4,16 @@
 
 A Shader Graph enables you to build shaders visually. Instead of hand writing code you create and connect nodes in a graph network. The graph framework gives instant feedback on the changes, and it’s simple enough that new users can become involved in shader creation.
 
-### Disclaimer
+## Disclaimer
 
-This repository is under active development. Everything is subject to change. The `master` branch is our current development branch and may not work on the latest publicly available version of Unity. Unless you intend to modify Shader Graph or want to try out the very latest and unsupported features, we recommend that you acquire Shader Graph through the Unity Package Manager.
+This repository is under active development. Everything is subject to change. The `master` branch is our current development branch and may not work on the latest publicly available version of Unity. Unless you intend to modify Shader Graph or want to try out the very latest and unsupported features, we recommend that you acquire Shader Graph through the Unity Package Manager:
```
Comment (jessebarker) [inline com.unity.shadergraph/README.md:9]: whitespace at the end of this line

File: com.unity.shadergraph/README.md [hunk 3]
```diff
@@ -4,17 +4,16 @@
...
+This repository is under active development. Everything is subject to change. The `master` branch is our current development branch and may not work on the latest publicly available version of Unity. Unless you intend to modify Shader Graph or want to try out the very latest and unsupported features, we recommend that you acquire Shader Graph through the Unity Package Manager: 
 
-## Instructions
-
-At the moment we recommend using Shader Graph through the [SRP repository](https://github.com/Unity-Technologies/Graphics), which has the Shader Graph submodule setup as a submodule. Otherwise you will not have any Master Node backends available and thus your shaders will be pink. This also ensure that you get a compatible set of render pipeline and Shader Graph versions. Otherwise, carry on with the following instructions.
+1. Open a Unity project.
```
Comment (jessebarker) [inline com.unity.shadergraph/README.md:11]: whitespace at the end of this line

File: com.unity.shadergraph/README.md [hunk 4]
```diff
@@ -4,17 +4,16 @@
...
+This repository is under active development. Everything is subject to change. The `master` branch is our current development branch and may not work on the latest publicly available version of Unity. Unless you intend to modify Shader Graph or want to try out the very latest and unsupported features, we recommend that you acquire Shader Graph through the Unity Package Manager: 
 
-## Instructions
-
-At the moment we recommend using Shader Graph through the [SRP repository](https://github.com/Unity-Technologies/Graphics), which has the Shader Graph submodule setup as a submodule. Otherwise you will not have any Master Node backends available and thus your shaders will be pink. This also ensure that you get a compatible set of render pipeline and Shader Graph versions. Otherwise, carry on with the following instructions.
+1. Open a Unity project. 
+2. Open the **Package Manager** window (**Window** &gt; **Package Manager**).
+3. In the **Package Manager** window, in the **Packages** menu, select **Unity Registry**.
```
Comment (jessebarker) [inline com.unity.shadergraph/README.md:13]: whitespace at the end of this line

File: com.unity.shadergraph/Documentation~/index.md [hunk 5]
```diff
@@ -6,6 +6,19 @@ Shader Graph enables you to build shaders visually. Instead of writing code, you
...
+
+For each release of Unity (alpha / beta / patch release) the graphics code is embedded within the main Unity installer. When you install the latest release of Unity, you also get the latest [Universal Render Pipeline (URP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest), [High Definition Render Pipeline (HDRP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest), Shader Graph, [Visual Effect (VFX) Graph](https://docs.unity3d.com/Packages.com.unity.visualeffectgraph@latest), and more.
+
+Tying graphics packages more closely to the main Unity release allows better testing to ensure that the graphics packages you use have been tested extensively with the version of Unity you have downloaded.
+
+You can also use a local copy or a custom version of the graphics packages by overriding them in the manifest file.
+
+For more information, see the following post on the forum: [SRP v11 beta is available now](https://forum.unity.com/threads/srp-v11-beta-is-available-now.1046539/).
+
```
Comment (jessebarker) [inline com.unity.shadergraph/Documentation~/index.md:24]: I didn't notice this at first, but it doesn't appear to like this empty line at the end.

File: com.unity.shadergraph/Documentation~/index.md [hunk 6]
```diff
@@ -6,6 +6,19 @@ Shader Graph enables you to build shaders visually. Instead of writing code, you
...
 
 Shader Graph package versions on Unity Engine 2018.x are *Preview* versions, which do not receive bug fixes and feature maintenance. To work with an actively supported version of Shader Graph, use Unity Engine 2019.1 or higher.
+
+## SRP packages are part of the core 
+
+With the release of Unity 2021.1, graphics packages are relocating to the core of Unity. This move simplifies the experience of working with new Unity graphics features, as well as ensuring that your projects are always running on the latest verified graphics code.
+
+For each release of Unity (alpha / beta / patch release) the graphics code is embedded within the main Unity installer. When you install the latest release of Unity, you also get the latest [Universal Render Pipeline (URP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest), [High Definition Render Pipeline (HDRP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest), Shader Graph, [Visual Effect (VFX) Graph](https://docs.unity3d.com/Packages.com.unity.visualeffectgraph@latest), and more.
```
Comment (ValGrimm-U3D) [inline com.unity.shadergraph/Documentation~/index.md:17]: "the graphics code is embedded within the main Unity installer. "
I propose
"graphics packages are included in the Unity installer."

and 
" [Visual Effect (VFX) Graph](https://docs.unity3d.com/Packages.com.unity.visualeffectgraph@latest), and more."
-->
" and [Visual Effect (VFX) Graph](https://docs.unity3d.com/Packages.com.unity.visualeffectgraph@latest) packages, among others."

File: com.unity.shadergraph/Documentation~/index.md [hunk 7]
```diff
@@ -6,6 +6,19 @@ Shader Graph enables you to build shaders visually. Instead of writing code, you
...
+
+Tying graphics packages more closely to the main Unity release allows better testing to ensure that the graphics packages you use have been tested extensively with the version of Unity you have downloaded.
```
Comment (ValGrimm-U3D) [inline com.unity.shadergraph/Documentation~/index.md:19]: I propose eliminating "more closely"

File: com.unity.shadergraph/Documentation~/index.md [hunk 8]
```diff
@@ -6,6 +6,19 @@ Shader Graph enables you to build shaders visually. Instead of writing code, you
...
+
+You can also use a local copy or a custom version of the graphics packages by overriding them in the manifest file.
```
Comment (ValGrimm-U3D) [inline com.unity.shadergraph/Documentation~/index.md:21]: "by overriding them in "
I propose
-->
"with an override in"

File: com.unity.shadergraph/README.md (hunk 2 above)
Comment (ValGrimm-U3D) [inline com.unity.shadergraph/README.md]: This looks like you deleted the disclaimer? That's right if so.

File: com.unity.shadergraph/README.md [hunk 9]
```diff
@@ -4,17 +4,16 @@
...
 
-## Instructions
-
-At the moment we recommend using Shader Graph through the [SRP repository](https://github.com/Unity-Technologies/Graphics), which has the Shader Graph submodule setup as a submodule. Otherwise you will not have any Master Node backends available and thus your shaders will be pink. This also ensure that you get a compatible set of render pipeline and Shader Graph versions. Otherwise, carry on with the following instructions.
+1. Open a Unity project. 
+2. Open the **Package Manager** window (**Window** &gt; **Package Manager**).
+3. In the **Package Manager** window, in the **Packages** menu, select **Unity Registry**. 
+4. Select either **Universal RP** or **High Definition RP**, depending on your project needs.
+    Unity installs your chosen render pipeline and Shader Graph. Shader Graph isn't compatible with Unity's built-in render pipeline.
```
[the remark this answers is not in the document]
Comment (ValGrimm-U3D) [reply com.unity.shadergraph/README.md]: But folks don't have to install Shader Graphfrom 21.2 forward, I thought?

In regard to Shader Graph compatibility with the Built-In Render Pipeline--Jesse Barker and Joshua Davis did some work in 21.2 that changes this, not sure it is being promoted yet, but not sure how this disclaimer should be phrased. 
See Built-In Target in https://unity3d.atlassian.net/browse/DOC-3229

File: com.unity.shadergraph/README.md [hunk 10]
```diff
@@ -4,17 +4,16 @@
...
+    Unity installs your chosen render pipeline and Shader Graph. Shader Graph isn't compatible with Unity's built-in render pipeline. 
 
-**Requires Unity 2018.1 Beta**
+## Instructions
 
-* Download the beta [here](https://unity3d.com/unity/beta)
-* Create a new project (or use an existing)
-* Clone branch `2018.1` into the `Assets` folder of your project, such that the repository is contained in a sub-folder of the `Assets` folder
-* Quick start guide can be found [here](https://forum.unity.com/threads/feedback-wanted-shader-graph.511960/)
+If you want to try out the latest features, we recommend using Shader Graph through the [SRP repository](https://github.com/Unity-Technologies/Graphics), which has the Shader Graph submodule setup as a submodule. Otherwise you will not have any Master Node backends available and thus your shaders will be pink. This also ensures that you get a compatible set of render pipeline and Shader Graph versions.
```
Comment (ValGrimm-U3D) [inline com.unity.shadergraph/README.md:19]: revised to:
"If you wish to try out the latest features, we recommend obtaining the most recent version of Shader Graph from the [SRP repository](https://github.com/Unity-Technologies/Graphics), which includes the Shader Graph submodule. If you do not install this submodule, you do not have any Master Node backends available and your shaders are invalid. (Invalid shaders appear pink.) Installing the submodule also ensures that you get a compatible set of render pipeline and Shader Graph versions."

What is a submoldule?
What else do you need to install?

File: com.unity.shadergraph/Documentation~/index.md [hunk 11]
```diff
@@ -6,11 +6,11 @@ Shader Graph enables you to build shaders visually. Instead of writing code, you
 
 For an introduction to Shader Graph, see [Getting Started](Getting-Started.md).
 
-Shader Graph is available through the Package Manager window in supported versions of the Unity Editor. 
+Shader Graph is available through the Package Manager window in supported versions of the Unity Editor. If you install a prebuilt Scriptable Render Pipeline (SRP) such as the [Universal Render Pipeline (URP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest) or the [High Definition Render Pipeline (HDRP)](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest), Unity automatically installs Shader Graph in your project.
```
Comment (ValGrimm-U3D) [inline com.unity.shadergraph/Documentation~/index.md:9]: Why "prebuilt"?

File: com.unity.shadergraph/Documentation~/index.md (hunk 6 above)
Comment (sarah-welton) [reply com.unity.shadergraph/Documentation~/index.md:17]: I didn't write this part - this was what was provided by the TPMs as something that apparently they want to be consistent across packages. It was just a cut and paste, so I don't know if we want to mess with it (though I admit I don't like how they wrote it, either)

File: com.unity.shadergraph/README.md (hunk 2 above)
Comment (sarah-welton) [reply com.unity.shadergraph/README.md]: So this whole chunk should disappear, instead?

File: com.unity.shadergraph/Documentation~/index.md (hunk 11 above)
Comment (sarah-welton) [reply com.unity.shadergraph/Documentation~/index.md:9]: Must have been another weird copy/paste. I can remove prebuilt.

File: com.unity.shadergraph/Documentation~/index.md (hunk 7 above)
Comment (sarah-welton) [reply com.unity.shadergraph/Documentation~/index.md:19]: Same as above - not written by me.

File: com.unity.shadergraph/Documentation~/index.md (hunk 8 above)
Comment (sarah-welton) [reply com.unity.shadergraph/Documentation~/index.md:21]: Same as above.

**Pull request closed without merging**